src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/ApplicationService.kt

```diff
@@ -799,7 +803,14 @@ class ApplicationService(
       situation = submitApplication.situation?.toString()
       inmateInOutStatusOnSubmission = inmateDetails?.custodyStatus?.name
       this.apArea = apArea
-      this.cruManagementArea = apArea.defaultCruManagementArea
+      this.cruManagementArea = if (featureFlagService.getBooleanFlag("cas1-womens-estate-enabled") &&
+        submitApplication.isWomensApplication == true
+      ) {
```

Comment:
it's probably worth checking with the UI team if we can make isWomensApplication non-null in the API. I'm pretty sure they'll always be populating it (even if it always currently 'false'). Can maybe do that on another ticket?

Reply:
Raised new ticket: https://dsdmoj.atlassian.net/browse/APS-1399 to work up this Sprint